src/utils/createTheme.js

```diff
@@ -0,0 +1,31 @@
+import { createMuiTheme } from '@material-ui/core/styles'
+import merge from 'lodash/merge'
+
+export default function createTheme(values = {}) {
+  let config = {}
+
+  merge(
```

Comment:
Can do without let: `const config =merge({},{...}, values)`

Reply:
Yes, it's just copypasta from RSF6. Looks like it can be removed though.